src/api/routes.py

```
@@ -0,0 +1,141 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from pydantic import BaseModel
+from typing import Dict, List, Optional, Any
```

Ruff (0.11.9)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)